pkg/internal/checkup/checkup.go

```diff
@@ -183,3 +201,89 @@ func randomizeName(prefix string) string {
 
 	return fmt.Sprintf("%s-%s", prefix, k8srand.String(randomStringLen))
 }
+
+func (c *Checkup) createAndRunTrafficGeneratorPod(ctx context.Context) error {
+	netAttachDef, err := c.client.GetNetworkAttachmentDefinition(
+		ctx,
+		c.namespace,
+		c.params.NetworkAttachmentDefinitionName)
+	if err != nil {
+		return err
+	}
```

Comment:
We can remove this call to get the NetworkAttachmentDefinition.
Its existence only matters to multus.